src/build.ts

```diff
@@ -267,11 +386,11 @@ export const BuildOutputs = {
   }): components['schemas']['JSONContentOutput'] {
     return {
       type: 'json-content',
-      ...(options?.plainText !== undefined && { plainText: options.plainText }),
-      ...(options?.structuredText !== undefined && { structuredText: options.structuredText }),
-      ...(options?.keyValuePairs !== undefined && { keyValuePairs: options.keyValuePairs }),
-      ...(options?.tables !== undefined && { tables: options.tables }),
-      ...(options?.language && { language: options.language }),
+      plainText: options?.plainText ?? true,
```

Comment:
I'd suggest defaulting to undefined here so we DWS can define what the defaults should be.